tests/unit/spring_boot_patch.py

```diff
@@ -182,10 +235,16 @@ def __init__(self) -> None:
         """Initialize the :class:`SpringBootPatch` instance."""
         self.container_mocks: typing.Dict[str, ContainerMock] = {}
         self.images: typing.Dict[str, OCIImageMock] = {}
-        self._patch = patch.multiple(
-            ops.model.Unit,
-            get_container=self._gen_get_container_mock(ops.model.Unit.get_container),
+        self._patches: typing.List[_patch] = []
+        self._patches.append(
+            patch.multiple(
+                ops.model.Unit,
+                get_container=self._gen_get_container_mock(ops.model.Unit.get_container),
+            )
         )
+        self._container_mock_callback: typing.Dict[
+            str, typing.Callable[[ContainerMock], typing.Any]
+        ] = {}
```

Comment:
Are the type hints required for class attributes ?

Reply:
Yes, it's required by `mypy`.